<launch>
<node type="laser_scan_assembler" pkg="laser_assembler"
    name="my_assembler">
<!-- <remap from="scan" to="laser_scan"/> -->
<param name="max_scans" type="int" value="1" />
<param name="fixed_frame" type="string" value="laser" />
 </node>
  <node type ="scan2pc.py" pkg="nav_sim" name="scan2pc" output="screen"/>
</launch>
